realm_test.spec fails on firefox

### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

admin/ui

### Describe the bug

 1) Realm tests
       should create realm with special characters:
     AssertionError: Timed out retrying after 30000ms: Expected to find content: '%22-61ef7587-4b98-459d-ae36-6792217a94b5' within the element: <li.pf-v5-c-menu__list-item> but never did.
      at goToRealm (webpack://@keycloak/keycloak-admin-ui/./cypress/support/pages/admin-ui/SidebarPage.ts:49:0)
      at  (webpack://@keycloak/keycloak-admin-ui/./cypress/e2e/realm_test.spec.ts:89:0)
      at __webpack_modules__</setRunnable/runnable.fn (http://localhost:8080/__cypress/runner/cypress_runner.js:138229:43)
      at callFn (http://localhost:8080/__cypress/runner/cypress_runner.js:156848:21)
      at __webpack_modules__</Runnable.prototype.run (http://localhost:8080/__cypress/runner/cypress_runner.js:156835:13)

### Version

main

### Regression

- [ ] The issue is a regression

### Expected behavior

Tests to run without issue

### Actual behavior

fails often

### How to Reproduce?

run the tests

### Anything else?

_No response_

--- a/js/apps/admin-ui/cypress/e2e/realm_test.spec.ts
+++ b/js/apps/admin-ui/cypress/e2e/realm_test.spec.ts
@@ -124,6 +124,7 @@ describe("Realm tests", () => {
   it("should create realm with special characters", () => {
     sidebarPage.goToCreateRealm();
     createRealmPage.fillRealmName(specialCharsName).createRealm();
+    sidebarPage.goToRealm("master");
 
     sidebarPage.goToRealm(specialCharsName);
     sidebarPage.goToClients();

